src/NServiceBus.AcceptanceTests/Routing/MessageDrivenSubscriptions/When_subscribing_to_a_base_event.cs

```diff
@@ -18,7 +18,7 @@ public Task Both_base_and_specific_events_should_be_delivered()
                     await session.Publish(new SpecificEvent());
                     await session.Publish<IBaseEvent>();
                 }))
-                .WithEndpoint<GeneralSubscriber>(b => b.When(async (session, c) => await session.Subscribe<IBaseEvent>()))
+                .WithEndpoint<GeneralSubscriber>(b => b.When(c => c.EndpointsStarted, async (session, c) => await session.Subscribe<IBaseEvent>()))
```

Comment:
I don't think this when condition is necessary, as the ATT framework will execute whens already after endpoints have been started. See ScenarioRunner line 193.